### PR TITLE
Adding Hebrew translation

### PR DESCRIPTION
**Pull request by yarons**

### CODE REVIEW

**Review by tchx84 — CHANGES_REQUESTED** [review]
Hey, thanks for the translation, but you also need to add this file name to `po/LINGUAS`

File: po/he.po [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=4; plural=(n==1 ? 0 : n==2 ? 1 : n>10 && n%10==0 ? "
+"2 : 3);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.desktop.in:4
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:4
+msgid "Flatseal"
+msgstr "Flatseal"
```
Comment (TheEvilSkeleton) [inline po/he.po]:
```suggestion
msgstr ""
```

File: po/he.po [hunk 2]
```diff
@@ -0,0 +1,545 @@
...
+#: src/widgets/aboutDialog.ui:11
+msgid ""
+"Flatseal is a graphical utility to review and modify permissions from your "
+"Flatpak applications."
+msgstr "Flatseal הוא כלי גרפי לסקירה ושינוי הרשאות של יישומי ה‏Flatpak שלך."
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:31
+msgid "Martin Abente Lahaye"
+msgstr "Martin Abente Lahaye"
```
Comment (TheEvilSkeleton) [inline po/he.po]:
```suggestion
msgstr ""
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]
Above these suggestions there is a comment saying to not translate this message.

**Comment by yarons** [comment]
Usually it breaks the statistics so I'm just copying the source but since you prefer it like that I don't care.

**Review by tchx84 — CHANGES_REQUESTED** [review]
hey @yarons , a few strings were added to the POT file sine you created this PR, mind updating as well?

**Comment by yarons** [comment]
Rebased, is that OK?

**Review by tchx84 — CHANGES_REQUESTED** [review]
@yarons yeah, it's now including the missing strings. The only thing left is to add `he` to the `LINGUAS` file.

**Comment by yarons** [comment]
@tchx84 Sure, done, thanks.

**Comment by tchx84** [comment]
Thank you!

**Pull request merged**